sw/device/silicon_creator/lib/drivers/mock_alert.h

```diff
@@ -0,0 +1,50 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_DRIVERS_MOCK_ALERT_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_DRIVERS_MOCK_ALERT_H_
+
+#include "sw/device/lib/testing/mask_rom_test.h"
+#include "sw/device/silicon_creator/lib/drivers/alert.h"
+
+namespace mask_rom_test {
+namespace internal {
+
+/**
+ * Mock class for alert.c.
+ */
+class MockAlert {
+ public:
+  MOCK_METHOD(rom_error_t, alert_configure,
+              (size_t, alert_class_t, alert_enable_t));
+  MOCK_METHOD(rom_error_t, alert_class_configure,
```

Comment:
Would you mind adding `alert_local_configure` (here and below)?

Reply:
done.